packages/e2e-tests-playwright/config/global-setup.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { chromium } from '@playwright/test';
+
+/**
+ * WordPress dependencies
+ */
+import { TestUtils } from '@wordpress/e2e-test-utils-playwright';
+
+export default async function setup() {
+	const browser = await chromium.launch();
+	const page = await browser.newPage( {
+		reducedMotion: 'reduce',
+	} );
+	const testUtils = new TestUtils( browser, page );
```

Comment:
```suggestion
	const testUtils = new TestUtils( page );
```
In theory you can get the browser like: `const browser = page.context().browser()`

Reply:
Oh awesome! Didn't know that.